docs/source/release/v6.12.0/Indirect/Bugfixes/38464.rst

```diff
@@ -0,0 +1 @@
+- Added a warning when a calibration workspace used in the ISISIndirectEnergyTransfer algorithm has zeroes for one or more spectra.
```

Comment:
Could we include a link to the documentation for `ISISIndirectEnergyTransfer` here?